vector/src/main/java/im/vector/app/features/home/room/list/RoomListViewEvents.kt

```diff
@@ -27,7 +27,7 @@ sealed class RoomListViewEvents : VectorViewEvents {
     data class Loading(val message: CharSequence? = null) : RoomListViewEvents()
     data class Failure(val throwable: Throwable) : RoomListViewEvents()
 
-    data class SelectRoom(val roomSummary: RoomSummary) : RoomListViewEvents()
+    data class SelectRoom(val roomSummary: RoomSummary, val isInviteAlreadyAccepted: Boolean?) : RoomListViewEvents()
```

Comment:
Should this new param be nullable? Can you confirm there is no need to have a default value?